### PR TITLE
Added doc for SUSPENDED state

### PR DESCRIPTION
**Pull request by eklein-okta**
@brianzuzga-okta @karlmcguinness-okta @georgekwon-okta 

This PR adds doc for the SUSPENDED state. The diagram is not updated yet. Wanted to get this out for review ASAP. Thanks.

### CODE REVIEW

**Comment by karlmcguinness-okta** [comment]
Seems straightforward.  Good with me

File: _source/docs/api/resources/users.md [hunk 1]
```diff
@@ -2027,6 +2029,86 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 ~~~
 
+### Suspend User
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-post"><span class="api-label">POST</span> /users/*:id*/lifecycle/deactivate</span>
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/users.md]: should this be `/users/*:id*/lifecycle/suspend` ?

File: _source/docs/api/resources/users.md [hunk 2]
```diff
@@ -2027,6 +2029,86 @@ HTTP/1.1 200 OK
...
+
+~~~http
+HTTP/1.1 200 OK
+Content-Type: application/json
+~~~
+
+### Unsuspend User
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-post"><span class="api-label">POST</span> /users/*:id*/lifecycle/deactivate</span>
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/users.md]: `/users/*:id*/lifecycle/unsuspend` ?

File: _source/docs/api/resources/users.md [hunk 3]
```diff
@@ -2027,6 +2029,86 @@ HTTP/1.1 200 OK
...
+{:.api .api-request .api-request-params}
+
+Parameter | Description  | Param Type | DataType | Required | Default
+--------- | ------------ | ---------- | -------- | -------- | -------
+id        | `id` of user | URL        | String   | TRUE     |
+
+##### Response Parameters
+{:.api .api-response .api-response-params}
+
+Returns an empty object.
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/users.md]: We should document the error status/code returned when user is in invalid status

File: _source/docs/api/resources/users.md [hunk 4]
```diff
@@ -2027,6 +2029,86 @@ HTTP/1.1 200 OK
...
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/users.md]: We should document the error status/code returned when user is in invalid status

Comment (brianzuzga-okta) [inline _source/docs/api/resources/users.md]: * Connected to rain.okta1.com (127.0.0.1) port 1802 (#0)
> POST /api/v1/users/00uih99uWd0ZoYzxM0g3/lifecycle/unsuspend HTTP/1.1
> User-Agent: curl/7.29.0
> Host: rain.okta1.com:1802
> Accept: application/json
> Content-Type: application/json
> Authorization: SSWS 00yTvtwAnrfonwRvXUMNgmfus_okh59So3oTsn4T4M
> 
< HTTP/1.1 400 Bad Request
< Server: Apache-Coyote/1.1
< x-okta-request-id: reqC18xuOJFQUmi_WzHuG0j6A
< X-Rate-Limit-Limit: 1200
< X-Rate-Limit-Remaining: 1197
< X-Rate-Limit-Reset: 1453845540
< P3P: CP="HONK"
< Set-Cookie: sid=""; Expires=Thu, 01-Jan-1970 00:00:10 GMT; Path=/
< Set-Cookie: JSESSIONID=B06AF903AE4F1EA9BA70A3EEC1E2E800; Path=/
< Cache-Control: no-cache, no-store
< Pragma: no-cache
< Expires: 0
< Content-Type: application/json
< Transfer-Encoding: chunked
< Date: Tue, 26 Jan 2016 21:58:08 GMT
< Connection: close
< 
* Closing connection 0
{"errorCode":"E0000001","errorSummary":"Api validation failed: unsuspendUser","errorLink":"E0000001","errorId":"oaeo8CBMHK2Q-CLYjufhhjWHQ","errorCauses":[{"errorSummary":"Cannot unsuspend a user that is not suspended"}]}

Comment (brianzuzga-okta) [inline _source/docs/api/resources/users.md]: So, that's a status code of 400.
Error code: E0000001

File: _source/docs/api/resources/users.md (hunk 3 above)
Comment (brianzuzga-okta) [inline _source/docs/api/resources/users.md]: Same as my remark below.

**Comment by eklein-okta** [comment]
Final changes and new diagram added. Good to go?

@brianzuzga-okta @karlmcguinness-okta

File: _source/docs/api/resources/users.md [hunk 5]
```diff
@@ -2027,6 +2029,86 @@ HTTP/1.1 200 OK
...
+{:.api .api-request .api-request-params}
+
+Parameter | Description  | Param Type | DataType | Required | Default
+--------- | ------------ | ---------- | -------- | -------- | -------
+id        | `id` of user | URL        | String   | TRUE     |
+
+##### Response Parameters
+{:.api .api-response .api-response-params}
+
+Returns an empty object. Returns an empty object. Passing an invalid `id` returns a `400 Bad Request` status code with error code `E0000001`.
```
Comment (brianzuzga-okta) [inline _source/docs/api/resources/users.md]: Invalid has two meanings here.  I thought the question Karl asked was what happened if you tried to suspend/unsuspend a valid user who happened to be in a state where that wasn't a valid transition.  That's a 400 + E0000001.  If you pass an invalid id, in the sense that it refers to an id that doesn't exist, you get a 404 + E0000007.  This is similar to other API calls.

**Comment by brianzuzga-okta** [comment]
:+1:

**Comment by eklein-okta** [comment]
Merging for preview release on Wednesday.

**Pull request merged**